test/SourceKit/NameTranslation/basic.swift

```diff
@@ -66,6 +67,7 @@ class MyDerived: FooClassDerived {
 // CHECK9: fooProperty11
 // CHECK10: init(nstanceFunc21:)
 // CHECK11: init(lassDerived2:)
+// CHECK11-1: FooClassDerived2
```

Comment:
When renaming from the `Foo` in `Foo(x: 1)`, we translate the type name instead of initializer name; and that's in consistence with our intended behavior. I cannot think of a situation when we need such lenience when renaming from Swift side.